src/javaExercises/oddEvenNumbers/OddEvenNumbers.java

```diff
@@ -0,0 +1,16 @@
+package javaExercises.oddEvenNumbers;
+
+import java.util.Arrays;
+
+public class OddEvenNumbers {
+    public static void findNumbers(int[] arr) {
+        System.out.println("Elements of the array are: " + Arrays.toString(arr));
+        System.out.println("In arrey are: " + Arrays.stream(arr).filter(x -> x % 2 == 0).count()
```

Comment:
please change it back to perform it in two separate methods with an return type and print element from outside, you can add a short method that will print any array

Reply:
Done